src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/tests/test_vm_commands.py

```diff
@@ -939,6 +928,47 @@ def body(self):
         finally:
             os.remove(config_file)
 
+class VMSSExtensionInstallTest(VCRTestBase):
```

Comment:
Changing this to a ResourceGroup test should not require a re-record as long as you don't change the resource group name (and make sure to delete myvmss3). It will just ensure that the next time it does need to be re-recorded it will be done the right way and not perpetuate a bad practice.

---
In reply to: [76100732](https://github.com/Azure/azure-cli/pull/737#discussion_r76100732) [](ancestors = 76100732,76078126)